torch_xla/csrc/tensor_methods.cpp

```diff
@@ -652,8 +652,14 @@ XLATensorPtr abs(const XLATensorPtr& input) {
 XLATensorPtr add(const XLATensorPtr& input, const XLATensorPtr& other,
                  const at::Scalar& alpha,
                  c10::optional<at::ScalarType> logical_element_type) {
+  SymIntElements sym_int_elements(other->GetIrValue());
```

Comment:
should we separate the static and dynamic case here?

Reply:
Since SymInt can be either static or dynamic, so I feel it's ok not separate the 2 cases. If CI is green, then this change should be good for the static case. wdyt?

Reply:
this will result in static code path to run unnecessary dynamic related code and make tracing slower. I'd prefer to separate the cases.

Reply:
done.

Btw, how can I see the tracing time? Is it a metric in `met.metrics_report()`?